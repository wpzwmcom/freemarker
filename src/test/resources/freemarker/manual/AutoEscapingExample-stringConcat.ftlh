<#ftl output_format="HTML"><#-- Needed for the Eclipse plugin -->
${"<h1>"?no_esc + "Foo & bar" + "</h1>"?no_esc}