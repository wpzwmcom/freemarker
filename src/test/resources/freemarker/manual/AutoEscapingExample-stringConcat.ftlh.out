<h1>Foo &amp; bar</h1>